CHANGES.rst

```diff
@@ -76,6 +76,8 @@ Improvements
 - Preserve non-ascii characters in file names (:issue:`4465`)
 - Allow resetting moderation state from registration management view
   (:issue:`4498`, thanks :user:`omegak`)
+- Create a new editing interface inside paper reviewing and add the option to
+  re-use accepted papers as editable files (:issue:`4489`).
```

Comment:
Just wondering if a changelog entry makes sense here since this is related to a new "major feature" (even though Editing is not listed there yet)

Reply:
Thought about the same but wasn't sure if you/others would agree. I can move it there.

Reply:
I think we'd have to write something else up there, since that changelog entry would need to mention the editing module itself, not just one small thing from it